com.jetbrains.CLion.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.jetbrains.CLion
+command: clion-desktop
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args: 
+  - --device=all
+  - --filesystem=home
```

Comment:
Why just home? Other JetBrains apps use host.

Reply:
Isn't less more? I guess I'm not sure what would be the rationale for sharing host rather home.

Reply:
Change applied. File system is set to host.